readme.md

```diff
@@ -1,5 +1,28 @@
 # Createjs Accessibility Module
 
+## Table of Contents
+- [Description](#description)
+  * [Standards](#standards)
+- [Install](#install)
+- [Usage](#usage)
+  * [Registering DisplayObject(s)](#registering-displayobject-s-)
+  * [Adding DisplayObject to Accessibility Tree](#adding-displayobject-to-accessibility-tree)
+  * [Registering the stage](#registering-the-stage)
+  * [Rendering the DOM](#rendering-the-dom)
+  * [Canvas resizing](#canvas-resizing)
+  * [Constraints](#constraints)
+  * [Events](#events)
+- [Test App](#test-app)
+  * [Usage](#usage-1)
+  * [Test Cases](#test-cases)
+  * [Adding to the test app](#adding-to-the-test-app)
+  * [Accessing the test app](#accessing-the-test-app)
+  * [Running the test app locally](#running-the-test-app-locally)
+- [Browsers Compatibility](#browsers-compatibility)
+- [License](#license)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
+
```

Comment:
❤️

Reply:
Is that okay to keep? 😅 😅

Reply:
my ❤️ was for the TOC itself, as far as if the generation link should be kept, I would say if TOC allows removal, we probably should remove or else leave the TOC out altogether